File: ssmMaven/ssm_maven_service/src/main/java/com/cnsdhh/ssmmaven/service/UserService.java
package com.cnsdhh.ssmmaven.service;

import com.cnsdhh.ssmmaven.pojo.User;

public interface UserService {

    // 根据username查询用户信息
    public User login(String username);

    // 添加登录用户信息
    public Integer register(User user);

}

File: ssmMaven/ssm_maven_web/src/main/webapp/static/js/user_service.js
// 为应用程序（app）定义一个服务：userService
app.service("userService", function ($http, $rootScope) {

    // 用户登录
    this.loginS = function (user) {
        return $http.post("/user/login", user);
    };

    // 注册用户
    this.registerS = function (user) {
        return $http.post("/user/register", user);
    }

});